rhp/v4/rpc_test.go

```diff
@@ -251,6 +247,15 @@ func TestFormContract(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	// validate contract signatures
```

Comment:
@n8maninger I added this because it seems like the RPC result didn't contain the renter signature. Which led to the host persisting the contract with an invalid signature as well and `RPCLatestRevision` returning the revision without signature. Causing the renter to be unable to broadcast the revision since it wasn't signed.